tests/mir-opt/pre-codegen/slice_iter.enumerated_loop.PreCodegen.after.panic-abort.mir

```diff
@@ -43,26 +43,6 @@ fn enumerated_loop(_1: &[T], _2: impl Fn(usize, &T)) -> () {
                             debug ptr => _9;
                             let mut _10: *const T;
                             scope 14 {
-                                scope 15 (inlined NonNull::<T>::new_unchecked::runtime::<T>) {
```

Comment:
For things in `mir-opt/pre-codegen`, see https://github.com/rust-lang/rust/blob/master/tests/mir-opt/pre-codegen/README.md where it notes that you can generally just bless them unless something's blowing up weirdly.  They're here to check the MIR for common patterns, and generally them getting smaller is good (so long as it's not because it's no longer inlining something important like a `slice::Iter::next`).